cmd/avalanche-network-runner/control/control.go

```diff
@@ -100,10 +101,16 @@ func newStartCommand() *cobra.Command {
 		"[optional] JSON string of map that maps from VM to its genesis file path",
 	)
 	cmd.PersistentFlags().StringVar(
-		&nodeConfig,
-		"node-config",
+		&defaultNodeConfig,
+		"default-node-config",
 		"",
-		"node config as string",
+		"default node config as string, applied to all nodes",
+	)
+	cmd.PersistentFlags().StringVar(
```

Comment:
We should give a better explanation of this flag's meaning. e.g. What happens if you don't pass it? What are the keys and values in the map the user passes? What if a flag is different than one in the default config?

Also, what would you think of using `StringToStringVar` instead of `StringVar`? i.e. the user does

```
--custom-node-configs node1={CONFIG GOES HERE},node2={CONFIG GOES HERE}
```

might be cleaner

Reply:
I opted to use `StringVar` to be consistent with the other map in this command, `customVMNameToGenesisPath,`

As for the better explanation - there is only so much which can go into the param description for the command...I'll give it a shot improving it, but tbh I think the extensive explanation should go to the docs.

Reply:
`I opted to use StringVar to be consistent with the other map in this command, customVMNameToGenesisPath`

I think it would probably make sense to use `StringToStringVar` there or any time we have a map from string to string. What do you think @gyuho ?

Reply:
With `StringVar` we keep the option to just use `--flagname $(echo config.json)` , which allows to maintain the config in a file which is easier to edit (especially for bigger configs).